auth/README.md

```diff
@@ -397,16 +397,47 @@ represented in the following diagram:
 
 ## UI customization
 
-To provide customization of the visual style of the activities that implement
-the flow, a new theme can be declared. Standard material design color
-and typography properties will take effect as expected. For example, to define
-a green theme:
+To use FirebaseUI Auth's sign-in flows, you must provide an `app_name` string and use the
+AppCompat color attributes in your app.
+
+For example, something like this goes in your `strings.xml`:
+```xml
+<resources>
+    <!-- ... -->
+    <string name="app_name">My App</string>
+</resources>
+```
+
+This goes in your `styles.xml`:
+```xml
+<style name="AppTheme" parent="FirebaseUI">
```

Comment:
The parent should be something like `Theme.AppCompat.DayNight.DarkActionBar` or `Theme.AppCompat.Light.DarkActionBar`, not `FirebaseUI`.